packages/cli/src/controllers/binary-data.controller.ts

```diff
@@ -38,7 +38,14 @@ export class BinaryDataController {
 				} catch {}
 			}
 
-			if (mimeType) res.setHeader('Content-Type', mimeType);
+			if (mimeType) {
+				res.setHeader('Content-Type', mimeType);
+
+				// Sandbox html files when viewed in a browser
+				if (mimeType === 'text/html' && action === 'view') {
+					res.header('Content-Security-Policy', 'sandbox');
```

Comment:
Isn't there a risque to break existing production code for user using html content with Javascript (or other features forbidden by the sandbox policy) ?

Reply:
viewing urls for binary data should never have been used a web-hosting mechanism.
If someone was doing this, then they were misusing binary-data. Since this has never been a part of our contract, sandboxing these urls won't be a breaking change.